src/main/resources/templates/appealCase.xml

```diff
@@ -0,0 +1,16 @@
+    <Appeal_Case>
```

Comment:
Do we want to be adding these xml files to the resources?

Reply:
yes for now, anyhow these will be cleared once test run is completed.